### PR TITLE
docs(config): update links of browser launchers

### PR DESCRIPTION
**Pull request by dschaller**

### CODE REVIEW

File: docs/config/03-browsers.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ Then, Karma will take care of auto-capturing these browsers, as well as killing
...
 ## Available browser launchers
-- [Chrome and Chrome Canary] (install karma-chrome-launcher)
-- [Firefox] (install karma-firefox-launcher first)
-- [Safari] (install karma-safari-launcher first)
-- [PhantomJS] (install karma-phantomjs-launcher)
-- [Opera] (install karma-opera-launcher first)
-- [IE] (install karma-ie-launcher first)
-- [SauceLabs] (install karma-sauce-launcher)
-- [BrowserStack] (install karma-browserstack-launcher)
+- [Chrome and Chrome Canary] (https://www.npmjs.com/package/karma-chrome-launcher)
```
Comment (dignifiedquire) [inline docs/config/03-browsers.md]: I think the space between the `[]` and the `()` needs to be removed

Comment (dschaller) [inline docs/config/03-browsers.md]: +1

**Comment by dignifiedquire** [comment]
Thanks :octocat:

**Comment by dignifiedquire** [comment]
Can you squash the two commits into one please?

**Comment by dschaller** [comment]
@dignifiedquire I squashed the commits but it looks like the build is failing because of something unrelated.
```
(::) failed steps (::)
Error: Step timed out after 5000 milliseconds
  at Timer.listOnTimeout (timers.js:92:15)
Failing scenarios:
test/e2e/timeout.feature:6 # Scenario: Broken Browser
```

**Comment by dignifiedquire** [comment]
Thanks :octocat:

**Comment by dignifiedquire** [comment]
Re failing tests, that happens sometimes when travis is slow :(

**Pull request merged**